k6/consistency/README.md

```
@@ -1,52 +1,415 @@
-```bash
-./k6/consistency/scripts/run-case.sh --mode async
-./k6/consistency/scripts/run-case.sh --mode outbox
+테스트의 목적은 결제 완료(Confirm) 이후 이벤트 발행 방식에 따른 **데이터 정합성(Consistency)과 시스템 신뢰성(Reliability)** 을 수치(정량)로 비교하여, **Transactional Outbox 도입의 정당성을 근거로 제시**하는 것이다.
+
+본 테스트는 **주문 상태 변경(E2E 비즈니스 처리)** 여부가 아니라,
+
+> **Kafka 장애 상황에서 이벤트 자체가 유실되었는지(Event Delivery Guarantee)** 를 순수하게 검증하는 데 목적이 있다.
+
+단순 비동기(Fire-and-Forget) 방식은 인프라 장애 상황에서 이벤트 유실(Data Loss)을 유발할 수 있으며, Outbox 패턴은 이를 DB에 안전하게 보관 후 재발행하여 **최종 유실 0(Eventual Consistency)** 로 복구함을 입증한다.
+
+### 🔎 검증 범위 (Scope)
```

markdownlint-cli2 (0.21.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)